judgments/views/detail.py

```diff
@@ -94,7 +94,6 @@ def detail(request, judgment_uri):
     # TODO: All references to `judgment` here need to be updated to the more general `document`
     context["judgment"] = judgment.content_as_html("")  # "" is most recent version
     context["judgment_uri"] = judgment.uri
-    context["judgment_title"] = judgment.name
```

Comment:
was redundant since page_title was the same as judgment_title and, judgment_title was only used once, so replaced that and removed this.